### ISSUE TITLE
lukewarm take: scroll down

### ISSUE THREAD

**Issue by BreadSpammer** [cs6413110/Pixel-Tanks#95]
### wat???

healing items are too meta defining. they are basically your only way of healing mid combat, and when your toolkit is on cooldown, and if you don't bring ANY (shield counts) you are at a HUGE disadvantage. they heavily outclass almost any item, because "why bring a bomb to destroy blocks, or a smoke bomb to cover yourself when u can just get 200 more HP from a healing item?".

### too much change?

this might change the game too much, but I say it might be in our interest to remove glu, tape, and shield, and replace them with natural regeneration, health increase, and healing on kill. (toolkit might die too depending if we make Regen too good. emergency shield still exists tho)

**Natural Regen:** if u don't get hit for 5 seconds, start healing at 1/25 of ur HP a second (20 HP, 1 bullet DMG), and if this is added, change the lower cooldowns perk to only work when Regen is active, and make it work rapidly (so u have items for the next fite)

**Healing on Kill**: give a free shield, and 150 health on kill. (note: don't make this a perk bc then it would become required to use lol)

**extra health**: health for players increased by 20% in order to not let ppl drop too fast.

### y tho???

removing healing items will make worse items get the chance to actually be used. with healing items, 2 of your item slots have to be dedicated to shield, and another healing item, which didn't leave much wiggle room for more unique items. you are then basically forced to use either dynamite and or reflect, bc it's the meta, and then you have no reason to use items such as airstrike, smoke bomb, and etc. without the healing items, you are more free to pick and choose bad items without being at a huge disadvantage.

healing items are also just abused a lot because with them, getting a toolkit off during a fight is near guaranteed if you play your cards right

adding Regen and healing on kill is vitally important for this change to work, because without them, you would have no Regen during, or after kills, due to the lack of healing items, which would lead to a lot of 3rd partying. 

### how will this change the meta???

basically every class gets equally overpowered due to the removal of survivability. it seems like tactical would become meta with it's one shot unimpeded by shield, but you would need to HIT that, and if you miss, ur kinda screwed. it would also be a lot harder to get to a good spot for an insta without the extra survivability. stealth would be pretty gud bc the 1 or 2 surprise hits mean a lot more, and getting away is way harder for other classes. builder would be another class that could safely heal, but their turrets would've a lot more prone to airstrikes, because they would be a lot more common. warrior would still be gud except against tactical, medic would probably need a nerf, and fire would be absolutley busted, bcuz u could pernamently stop their regen

**Comment by LostKing73** [comment]
ai slop

**Comment by LostKing73** [comment]
warrior would need some sort of buff, but ig

**Comment by BreadSpammer** [comment]
its not ai lol i wasted my 6th hour for this essay

**Comment by LostKing73** [comment]
ik, but like ehhhhhhhhh

**Comment by LostKing73** [comment]
cold take: like no lol?

**Comment by cs6413110** [comment]
Ok, so, first...

> healing items are also just abused a lot because with them, getting a toolkit off during a fight is near guaranteed if you play your cards right

Actually you can die toolkitting at full.

> basically every class gets equally overpowered due to the removal of survivability.

Changing items won't affect or balance classes, except medic. And it will only buff medic.

But....

It will make players and AI more consistent. I'll actually get to use 4 items instead of 2.

**Comment by LostKing73** [comment]
lukewarm take: remove sheild as an item. still leave it for that one perk, but since shield is the ONE item you MUST have, remove. lets u get another item, and like shield covers up all my beautiful cosmets anyways so like git it outta here, yes?

**Comment by BreadSpammer** [comment]
ye true, but like tape/glu are also kinda required. if u dont have them u have 0 way to regen other than toolkit. and toolkit is unreliable without said items. also this just makes tanks a lot squishier, and makes game fasterpace so maybe

**Comment by LostKing73** [comment]
is vote so vote

**Comment by LostKing73** [comment]
@cs6413110 @BreadSpammer @CelestialKnight7

**Comment by CelestialKnight7** [comment]
everything on this issue is worthy of downvoting to oblivion
shield is fine
healing is fine
shut

**Comment by LostKing73** [comment]
ok, just wanted u to vote. i just think shield covers ur tank too much
